.github/workflows/python.yml

```diff
@@ -167,3 +167,61 @@ jobs:
           workspaces: python
       - uses: ./.github/workflows/build_windows_wheel
       - uses: ./.github/workflows/run_tests
+  aws-integtest:
+    timeout-minutes: 45
+    runs-on: "ubuntu-22.04"
+    defaults:
+      run:
+        shell: bash
+        working-directory: python
+    services:
```

Comment:
FYI For local testing of S3 + dynamodb, we've found localstack is easy to setup:

https://github.com/delta-io/delta-rs/blob/da173096e56e08ce38898f62d9520fd4082e15d3/docker-compose.yml#L3-L18

Reply:
will migrate to localstack soon #1241